src/storybook/helpers.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent } from 'react'
+
+import { action } from '@storybook/addon-actions'
+
```

Comment:
Same here.

Reply:
```suggestion
```

Reply:
I guess it's a habit of mine. I like to visually separate the imports into external and project-defined modules. I don't mind to drop this, though.